numpy/lib/tests/test_function_base.py

```diff
@@ -1641,6 +1641,23 @@ def test_simple(self):
         # check that unwrap maintains continuity
         assert_(np.all(diff(unwrap(rand(10) * 100)) < np.pi))
 
+    def test_minmax(self):
+        # check that unwrap removes jumps greater that 255
+        assert_array_equal(unwrap([1, 1 + 256], min_val=0, max_val=255), [1, 2])
+        # check that unwrap maintains continuity
+        assert_(np.all(diff(unwrap(rand(10) * 1000, min_val=0, max_val=255)) < 255))
+        # check simple case
+        simple_seq = np.linspace(0, 512, 6)
```

Comment:
This is a bit of a weird sequence, `array([  0. , 102.4, 204.8, 307.2, 409.6, 512. ])`. Was that deliberate?

Reply:
it was originally longer but 6 was long enough to show the point but admittedly 0-512 with 6 steps is rather ugly